week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -72,7 +86,26 @@ function insertShoppingList(shoppingList) {
  * The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com
  */
 function insertBooks(books) {
-  //Write your code in here
+  let bookslist = document.createElement('ul')
+  bookslist.style.display = 'flex'
+  let bookImgSrc = ['./img/1.jpg', './img/2.jpg', './img/3.jpg']
+  books.forEach((p, i) => {
+    let book = document.createElement('li')
+    book.style.width = '350px'
+    book.style.margin = '10px'
+    book.style.listStyle = 'none'
+    p.alreadyRead ? book.style.backgroundColor = 'green' : book.style.backgroundColor = 'red'
+    let bookName = document.createElement('p')
+    bookName.innerText = `${p.title}-${p.author}`
+    let bookImg = document.createElement('img')
+    bookImg.style.width = '200px'
+    bookImg.style.margin = '5px'
+    bookImg.src = bookImgSrc[i]
+    book.appendChild(bookName)
+    book.appendChild(bookImg)
+    bookslist.appendChild(book)
+  })
```

Comment:
Perfect again
how about adding the link to the image (url maybe)
more image attributes (alt, href, target)